openpower/package/hostboot/hostboot.mk

```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 HOSTBOOT_VERSION_BRANCH_MASTER_P8 ?= f2b28bf6eee57757be45c45e514d2542d72aa350
-HOSTBOOT_VERSION_BRANCH_MASTER ?= b9fd10a99329adb4093c453bf9f80b05a6f31fa3
+HOSTBOOT_VERSION_BRANCH_MASTER ?= eb217e512338e8d2182762cbe97f55d4759e799e
```

Comment:
I'm worried about seeing a version update here because the Hostboot team already has an auto-update job they run.  Is this really needed?